### ISSUE TITLE
Significant performance drops in-game [BUG]

### ISSUE THREAD

**Issue by RebelLion420** [lacymorrow/crossover#332]
Whenever I have Crossover enabled over my game in Windowed Fullscreen mode, my FPS tanks from ~170 down to ~50 and the game itself looks extra choppy. I tried disabling GPU acceleration and running it as Admin but nothing changes. As soon as I close Crossover or Alt+Tab out of the game the lag disappears. Is there anything I can do to fix it?

**To Reproduce**

Run Crossover, launch the game and FPS is noticeably bad. You can restart Crossover and the issue persists as long as the game is focused

**Screenshots**

**Versions (please complete the following information):**
 - OS: Windows 11
 - CrossOver Version 3.3.4
 - Game: The FINALS

**Comment by RebelLion420** [comment]
So, I quickly found out this is just caused by V-Sync. I tried an overlay using the Game Bar and had the same performance issue. Did some more research and found a post on about the Steam Overlay causing similar lag issues, and the possible cause being V-Sync. I disabled both the driver V-Sync and display Adaptive Sync and the issue is gone. Not sure if this is already known but I'll leave this here just in case
